engines/scumm/imuse_digi/dimuse_track.cpp

```diff
@@ -171,7 +171,7 @@ void IMuseDigital::startSound(int soundId, const char *soundName, int soundType,
 				track->regionOffset -= track->regionOffset >= (track->feedSize / _callbackFps) ? (track->feedSize / _callbackFps) : 0;
 			}
 		}
-		if (_vm->_game.id == GID_CMI) {
+		if (_vm->_game.id == GID_CMI && (track->volGroupId == IMUSE_VOLGRP_MUSIC)) {
```

Comment:
I don't know why this has been duplicated, it should't have. This is the same fix as commit 3d9a8862046acb84feb9bf6256c21b492cd67ff0

Reply:
you need work on separated branch

Reply:
Whoops, sorry, you were right. I probably opened this branch concurrently to the other one you merged an hour ago.